modules/learn/pages/clusters-and-availability/xdcr-filtering.adoc

```diff
@@ -50,14 +50,17 @@ You can enclose a range of characters in square brackets, to match against all o
 | Matches any alphanumeric character.
 
 | `^`
-| Matches beginning of input.
-If the multi-line flag is set to `true`, also matches immediately after a line-break character.
-For example, `/^A/` does not match the '[.code]``A``' in "[.code]``an A``", but does match the '[.code]``A``' in "[.code]``An E``".
+| Matches the beginning of input.
+For example, `/^a/` matches `abb` and `ab`; but does _not_ match `ba`, `bab`, or `bba`.
+If the multi-line flag is set to `true`, `^` matches the start of _any line_ within the input string, instead of the beginning of the entire string.
```

Comment:
I realise that it's not the focus of the PR, but is this correct?  Not sure a multi-line flag applies in this context... perhaps something that was carried over from docs on the regex lib?

Reply:
Yes, could well be. I have removed.